var/spack/repos/builtin/packages/parmetis/package.py

```diff
@@ -63,8 +64,8 @@ def install(self, spec, prefix):
 
         # FIXME : Once a contract is defined, MPI compilers should be retrieved indirectly via spec['mpi'] in case
         # FIXME : they use a non-standard name
-        options.extend(['-DGKLIB_PATH:PATH={metis_source}/GKlib'.format(metis_source=metis_source),
-                        '-DMETIS_PATH:PATH={metis_source}'.format(metis_source=metis_source),
+        options.extend(['-DGKLIB_PATH:PATH={metis_source}/GKlib'.format(metis_source=metis_source), # still need headers from METIS source, and they are not installed with METIS. shame...
```

Comment:
if you look below, what was patched is `metis_header = join_path(metis_source, 'include', 'metis.h')` and not `GKLIB`. Since we provide `metis` path to an already installed `metis` with, supposedly, those `idx64` and so on, it should be ok.